esx_service/utils/vmdk_utils.py

```diff
@@ -94,7 +94,7 @@ def vmdk_is_a_descriptor(filepath):
                 line = f.readline()
                 return line.startswith('# Disk DescriptorFile')
         except:
-            logging.warning("Failed to open %s for descriptor check", filepath)
+            logging.exception("Failed to open %s for descriptor check", filepath)
```

Comment:
Since we aren't catching a specific exception any idea what message we get in this case? Can we catch a generic exception that will get logged?